packages/server/graphql/mutations/helpers/generateRetroSummary.ts

```diff
@@ -0,0 +1,41 @@
+import yaml from 'js-yaml'
+import getKysely from '../../../postgres/getKysely'
+import OpenAIServerManager from '../../../utils/OpenAIServerManager'
+import {DataLoaderWorker} from '../../graphql'
+import canAccessAI from './canAccessAI'
+import {transformRetroToAIFormat} from './transformRetroToAIFormat'
+
+export const generateRetroSummary = async (
```

Comment:
+1 I think the name is misleading. I would assume this function does just generate a summary for a meeting and returns it. Instead it updates the table.
I would prefer if we'd change it to `updateRetroSummary` and also remove the return type, so it's less likely to be used wrongly.

Reply:
I feel like `updateRetroSummary` suggests it already exists and we're updating it, but we're creating a new summary here. 

Maybe `addRetroSummary` instead of `generateRetroSummary`? I like that `generate` describes that this is where it's being created.